frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -51,6 +54,8 @@ export const createKnativeService = (
       limitUnit: memoryLimitUnit,
     },
   } = limits;
+  const defaultLabel = getAppLabels(name, applicationName, imageStreamName);
+  delete defaultLabel.app;
```

Comment:
@sahil143 `app.kubernetes.io/instance` we still have right? this will create an issue in case of any config changes as we discussed yesterday. WDYT?

Reply:
@invincibleJai Yes, We still have the instance label. we haven't decided yet if we want to remove it or what. I created a separate issue: https://jira.coreos.com/browse/ODC-1471

@christianvogt ^^

Reply:
@sahil143 okay, we need to handle this as separate issue as with this change we'll see different workloads with same name in case of config changes.